tests/project/test_brownie_config.py

```diff
@@ -98,9 +98,6 @@ def test_rpc_project_cmd_settings(devnetwork, testproject, config, settings_proj
     assert web3.eth.chainId == 666
     assert web3.net.version == "777"
 
-    # Test if evm version is returned properly
-    assert devnetwork.rpc.evm_version() == cmd_settings_proj["evm_version"]
```

Comment:
why not check this anymore?

Reply:
It was already challenging with only ganache because if the instance was already running, we had to extract it from the commandline args.  In generalizing the RPC we'd now have to find a way to determine this from multiple clients, and I feel like the ++ to complexity weighed against the frequency that somebody actually has a fail from this.. simpler to just remove the check.